tensorflow/core/platform/default/build_config.bzl

```diff
@@ -4,8 +4,8 @@ load("//google/protobuf:protobuf.bzl", "cc_proto_library")
 load("//google/protobuf:protobuf.bzl", "py_proto_library")
 
 # configure may change the following lines.
-CUDA_VERSION = '7.0'
-CUDNN_VERSION = '6.5'
+CUDA_VERSION = '.7.0'
```

Comment:
1) isn't configure *going* to change these lines?

2) Should the default be empty, or at least, default to the latest version we know to work?